src/components/TripInfo.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from "react";
```

Comment:
Maybe in this components too one state hook would make code more clear...  mentioned in the comment above :) `src/components/TravelInfo.tsx`